src/SonataUserBundle.php

```diff
@@ -58,4 +59,19 @@ public function registerFormMapping(): void
             ]);
         }
     }
+
+    /**
+     * We MUST remove this method when support for "friendsofsymfony/user-bundle" is dropped
+     * or adapted to work with "doctrine/common:^3".
+     */
+    private function createDoctrineCommonBackwardCompatibilityAliases(): void
```

Comment:
What about making this method public static in order to call it in the custom_boostrap ?

Reply:
Then it would be bc break to remove it, IMO not a good idea.